drivers/serial/Kconfig.nrf5

```diff
@@ -32,23 +32,23 @@ config UART_NRF5_FLOW_CONTROL
 
 config UART_NRF5_GPIO_TX_PIN
 	int "TX Pin Number"
-	range 0 31
+	range 0 48
```

Comment:
use range 0 48 if SOC_NRF52840

Reply:
And it should be 0 47. nRF52840 has 48 GPIOs, not 49.

Reply:
done